tidb-performance-tuning-config.md

```diff
@@ -88,26 +89,62 @@ concurrent-send-snap-limit = 64
 concurrent-recv-snap-limit = 64
 snap-io-max-bytes-per-sec = "400MiB"
 
+[quota]
+background-read-bandwidth = "400Mib"
+
 [pessimistic-txn]
 in-memory-peer-size-limit = "32MiB"
 in-memory-instance-size-limit = "512MiB"
 
+[rocksdb]
+rate-bytes-per-sec = "600MiB"
+max-manifest-file-size = "256MiB"
 [rocksdb.titan]
 enabled = true
 [rocksdb.defaultcf.titan]
 min-blob-size = "1KB"
 blob-file-compression = "zstd"
 
+[storage]
+scheduler-pending-write-threshold = "512MiB"
 [storage.flow-control]
 l0-files-threshold = 60
+soft-pending-compaction-bytes-limit = "1TiB"
+hard-pending-compaction-bytes-limit = "2TiB"
 ```
 
 | Configuration item | Description | Note |
 | ---------| ---- | ----|
 | [`concurrent-send-snap-limit`](/tikv-configuration-file.md#concurrent-send-snap-limit), [`concurrent-recv-snap-limit`](/tikv-configuration-file.md#concurrent-recv-snap-limit), and [`snap-io-max-bytes-per-sec`](/tikv-configuration-file.md#snap-io-max-bytes-per-sec) | Set limits for concurrent snapshot transfer and I/O bandwidth during TiKV scaling operations. Higher limits reduce scaling time by allowing faster data migration. | Adjusting these limits affects the trade-off between scaling speed and online transaction performance. |
+| [`quota.background-read-bandwidth`](/tikv-configuration-file.md#background-read-bandwidth-new-in-v620) | Sets a soft limit on read bandwidth for background transactions and Coprocessor operations. Recommended to configure at 40% of disk bandwidth to prevent interference with foreground traffic. | Default: unlimited. Without this limit, analyze jobs on large tables may significantly degrade application performance. |
```

Comment:
```suggestion
| [`quota.background-read-bandwidth`](/tikv-configuration-file.md#background-read-bandwidth-new-in-v620) | Set a soft limit on read bandwidth for background transactions and Coprocessor operations. It is recommended that you configure at 40% of disk bandwidth to prevent interference with foreground traffic. | Default value: `0KiB` (which means no limit). Without this limit, analyze jobs on large tables might significantly degrade application performance. |
```